pubspec.yaml

```diff
@@ -46,7 +46,7 @@ dependencies:
   go_router: ^13.0.0
   flutter_svg: ^2.0.4
   flutter_blue_plus: ^1.15.5
-  intl: ^0.18.1
+  intl: '>=0.18.0 <1.0.0'
```

Comment:
are you sure about this?

Reply:
Yes - there has been some discussion on the `intl` package and problem with Flutter - not solved yet, so this is the current work-around.